

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>NGS data of Maize</title>
    <meta name="description" content="description">
    <meta name="author" content="Jinliang Yang">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles change to Yihui style -->
    <!-- link rel="stylesheet" href="/media/css/style.css"  -->
    <link rel="stylesheet" href="/media/css/highlight.css">
    <link href="/assets/themes/twitter/bootstrap/css/bootstrap.2.2.2.min.css" rel="stylesheet">
    <link href="/assets/themes/twitter/bootstrap/css/style.css" rel="stylesheet">
    <link href="/assets/themes/twitter/css/style.css?body=1" rel="stylesheet" type="text/css" media="all">

    <!-- Le fav and touch icons -->
  <!-- Update these with your own images
    <link rel="shortcut icon" href="images/favicon.ico">
    <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
    <link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png">
  -->

    <!-- atom & rss feed -->
    <link href="/atom.xml" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="/rss.xml" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">

  </head>

  <body>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container-narrow">
          <a class="brand" href="/">Jinliang Yang</a>
          <ul class="nav">
            
            
            


  
    
      
    
  
    
      
      	
      	<li><a href="/archive">Archive</a></li>
      	
      
    
  
    
      
    
  
    
      
      	
      	<li><a href="/categories">Categories</a></li>
      	
      
    
  
    
      
    
  
    
      
      	
      	<li><a href="/pages">Pages</a></li>
      	
      
    
  
    
      
    
  
    
      
    
  
    
      
      	
      	<li><a href="/tags">Tags</a></li>
      	
      
    
  
    
      
    
  
    
      
    
  



          </ul>
        </div>
      </div>
    </div>

    <head>
      <meta charset="utf-8">
      <title>Jinliang Yang | 杨金良</title>
      <meta name="author" content="Jinliang Yang | 杨金良"/>
      <meta name="viewport" content="width=device-width; initial-scale=1.0">
      <link rel="stylesheet" type="text/css" href="/media/css/home.css">
      <style type="text/css">
      html, body {
        background-color: #F8F8F8;
      }
      .container > footer p , .postheader > a{
        text-align: center;
      }
      .date {
        text-align: center;
        height: 30px;
        color:#bfbfbf;
        margin-bottom: 10px;
      }
      /* Specify class=linenums on a pre to get line numbering */
      ol.linenums {
        margin: 0 0 0 33px; /* IE indents via margin-left */
      }
      ol.linenums li {
        padding-left: 12px;
        color: #bebec5;
        line-height: 18px;
        text-shadow: 0 1px 0 #fff;
      }
      </style>
      <script type="text/javascript">
        var _gaq = _gaq || [];
        _gaq.push(['_setAccount', 'UA-1988641-2']);
        _gaq.push(['_trackPageview']);
        (function() {
          var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
          ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
          var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
        })();
      </script>
      <!-- weibo -->
      <script src="http://tjs.sjs.sinajs.cn/open/api/js/wb.js" type="text/javascript" charset="utf-8"></script>
    </head>
    <body>
      <div class="container">
      <ul class="nav pills">
      <li><a href="/">Home</a></li>
      <li><a href="/cn/">中文</a></li>
      <li><a href="/en/">English</a></li>
      <li><a href="/about/">About</a></li>
      <li><a href="/vitae/">VITAE</a></li>
    </ul>
  </div>
  </body>









    <div class="container-narrow">

      <div class="content">
        

<div class="page-header">
  <h1>NGS data of Maize </h1>
</div>

<div class="row-fluid post-full">
  <div class="span12">
    <div class="date">
      <span>05 March 2014</span>
    </div>
    <div class="content">
      
<p>We are now in the Big Data era, where “the data scientist will become the most sexy job in the world” said by <a href="http://www.nytimes.com/2013/04/14/education/edlife/universities-offer-courses-in-a-hot-new-field-data-science.html?pagewanted=all&amp;_r=0">NYT</a>. And the IT world frequently mentioned “90% of the data in the world today has been created in the last two years alone”. It seems like a strong wave of data explosion is coming!</p>

<p>Amazing! However, I feel like we have nothing to do with it until recently when I checked the NCBI SRA database for a talk. I found that, we, the maize community, are right in the midst of the Big Data explosion. According to the database report, while after doing some accessions lookup, in the past several years, especially the last two years, about 13,000 Giga base-pairs were deposited from the maize community. That is about 6,500 depth of coverage of the maize genome if roughly assuming the maize genome is 2 Giba bases in size. This estimation not includes the on-going projects and non-submitted datasets. More importantly, this is happening just five years after the accomplishment of the maize B73 reference genome project. If we review the complete datasets in the SRA database, maize NGS data actually only accounted for &lt; 0.1%. Until the end of Feb. 2014, more than 2,300 Tera bases reside there. The “90% <strong>genome sequencing data</strong> were generated in the past two years alone”. See, IT’s data theory could be perfectly aligned to the sequencing industry (see the figure below).  </p>

<p><img src="http://i.imgur.com/Mb5Z1Yi.png" alt="alt text" /></p>

<p>As the data explosion, NGS data’s storage, sharing and publishing become a worldwide challenge. To overcome it, ~150 academic institutions and private companies went together to join the <a href="http://genomicsandhealth.org/">Global Alliance for Genomics and Health</a>. While, <a href="http://googleresearch.blogspot.co.uk/2014/02/google-joins-global-alliance-for.html">Google Genomics</a> is one of them. As the joining of some IT tycoons, <strong>genomics</strong> now become a multi-disciplinary challenge. <strong>Genomics</strong> and its related <strong>Bioinformatics</strong> are upgraded to their 2.0 version. I never thought that, trained as a geneticist, I may one day work for Google. Good, bad or ironic? It might be too optimistic. There is a long way to go and the industries are actually more interested in the human health related area. As a relative small research community for maize or plant genomics studies, we are less attractive from industry’s point of view. </p>

<p>Here, I will try to interpret the maize NGS data a little. This kind of study may be extended to other crop species later. After a close look, the data presented their own story. This kind of story might not be extracted from the papers alone. Below, I listed the top 10 maize projects according to their data output.</p>

<p><img src="http://i.imgur.com/01JWlMz.png" alt="alt text" /></p>

<p>From the above figure, it seems like the maize genetic studies were expanding from their traditional research areas. Thanks to the continuous declining of the sequencing price, many genome wide studies become possible, such as geno-pheno association, transcriptome, methylome, etc. At the same time, the resources and talents were under “exon reshuffling” during this process. Some Chinese groups, including those from CAU, CAAS and Huazhong Agricultural University are rising. </p>

<hr />
<p>Top 10 projects （<a href="/data/SRA_maize_030614.csv">download complete table</a>）</p>

<table>
  <thead>
    <tr>
      <th><strong>Order</strong></th>
      <th><strong>Center</strong></th>
      <th style="text-align: center"><strong>Size (Gb)</strong></th>
      <th><strong>Summary</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>CAAS</td>
      <td style="text-align: center">2426</td>
      <td>transcriptome</td>
    </tr>
    <tr>
      <td>2</td>
      <td>MSU-BUELL</td>
      <td style="text-align: center">1247</td>
      <td>Pan Transcriptome</td>
    </tr>
    <tr>
      <td>3</td>
      <td>CAU</td>
      <td style="text-align: center">1224</td>
      <td>Genome Re-sequencing</td>
    </tr>
    <tr>
      <td>4</td>
      <td>CSHL</td>
      <td style="text-align: center">964</td>
      <td>Maize HapMap II</td>
    </tr>
    <tr>
      <td>5</td>
      <td>Academia Sinica</td>
      <td style="text-align: center">626</td>
      <td>Transcriptome of maize embryonic leaves</td>
    </tr>
    <tr>
      <td>6</td>
      <td>University of Minnesota</td>
      <td style="text-align: center">619</td>
      <td>Transcriptome of IBM RILs</td>
    </tr>
    <tr>
      <td>7</td>
      <td>CSHL</td>
      <td style="text-align: center">572</td>
      <td>Methylome</td>
    </tr>
    <tr>
      <td>8</td>
      <td>ISU</td>
      <td style="text-align: center">512</td>
      <td>Zeanome</td>
    </tr>
    <tr>
      <td>9</td>
      <td>Cornell University</td>
      <td style="text-align: center">493</td>
      <td>Breeding efforts in Africa</td>
    </tr>
    <tr>
      <td>10</td>
      <td>MSU-BUELL</td>
      <td style="text-align: center">450</td>
      <td>Gene Expression of vitamin biosynthesis</td>
    </tr>
  </tbody>
</table>

<hr />

<p>In the U.S., from the above summary table of the sequencing data, I learned that different labs have their own focus. The <strong>lab</strong> here, sometimes, should also refer to the efforts from their collaborators. Among these labs, Buckler’s group at Cornell, after finishing the zea mays HapMap project, was now aiming to sequencing every species of maize in the world using Genotyping-By-Sequencing (GBS). Recently, Buell’s lab at MSU released a large set of transcriptomic data. This group may lead the maize transcriptome analysis in the future. A group at CSHL did a lot of Methylome study. Another group at CSHL, Jackson’s lab, focused on the development studies, their efforts of transcriptomic sequencing at various developmental stages may eventually lead to a developmentome (I invented this term) some day. Other research subjects, such as ISU’s Zeanome and PAV, UC Davis R-I group’s domestication and evolution and University of Minnesota’s IBM RIL eQTL mapping, et. al.. </p>

<p>Sequencing alone could not solve the biological problems until applied with sophisticated analyses. When genetics meets Big Data, the analysis itself become a problem. Traditional training did not prepared genetists with big data skills. In the future, the one who mastering the data tools and making sense of data would be more likely to success I guess. Genetics is now not limited to pipetting, pollinating or cloning, coding skills should be added to our toolkit as well.</p>

<p>Big Data era in maize is coming, are you ready yet?</p>


    </div>

  
    <ul class="tag_box inline">
      <li><i class="icon-folder-open"></i></li>
      
      


  
     
    	<li><a href="/categories.html#BigData of Maize-ref">
    		BigData of Maize <span>1</span>
    	</a></li>
    
  


    </ul>
    

  
    <ul class="tag_box inline">
      <li><i class="icon-tags"></i></li>
      
      


  
     
    	<li><a href="/tags.html#NGS-ref">NGS <span>7</span></a></li>
     
    	<li><a href="/tags.html#BigData-ref">BigData <span>1</span></a></li>
    
  



    </ul>
    

    <hr>
    <div class="pagination">
      <ul>
      
        <li class="prev"><a href="/lessons/2011/12/29/jekyll-introduction" title="Jekyll Introduction">&larr; Previous</a></li>
      
        <li><a href="/archive.html">Archive</a></li>
      
        <li class="next"><a href="/journal%20club/2014/04/09/Journal-Club-2" title="Journal Club（2）">Next &rarr;</a></li>
      
      </ul>
    </div>
    <hr>
    


  <div id="disqus_thread"></div>
<script type="text/javascript">
    var disqus_developer = 1;
    var disqus_shortname = 'yangjl'; // required: replace example with your forum shortname
    
    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">blog comments powered by <span class="logo-disqus">Disqus</span></a>




  </div>
</div>


      </div>
      <hr>
      <footer align="center">
         <p><small>Powered by <a href="https://github.com/mojombo/jekyll">Jekyll</a> on GitHub | Copyright 2016 by <a href="http://yangjl.com">Jinliang Yang</a> | <span class="label label-info">2016-09-09 14:28:05</span></small></p>
      </footer>

    </div>

    
  </body>
</html>
